documentation/Security Descriptor.asciidoc

```diff
@@ -213,14 +217,25 @@ The object ACE data structure is variable of size and consists of:
 | Offset | Size | Value | Description
 | 0 | 4 | | Access rights flags (ACCESS_MASK) +
 See section: <<access_rights_flags,Access rights flags (ACCESS_MASK)>>
-| 4 | 4 | | Flags
-| 8 | 16 | | Object type class identifier +
-Contains a GUID
-| 24 | 16 | | Inherited object type class identifier +
-Contains a GUID
+| 4 | 4 | | Object flags
+| 8 | 16 (opt.) | | Object type class identifier +
+Contains a GUID. This field is only present if the ACE_OBJECT_TYPE_PRESENT object flag is set.
```

Comment:
Is it present (as in allocated for in the structure) or set (as in the structure member is used)?